anime-flutter/anime_galaxy/lib/anime_setting/ui/widget/circular_setting/circular_image.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+
+class CircularImage extends StatelessWidget {
```

Comment:
I think we should put repated widget in common folder instead of rewriting it in each module